README.md

```diff
@@ -79,7 +81,7 @@ There are some guidelines which will make applying PRs easier for us:
 + No tabs! Please use spaces for indentation.
 + Respect the code style.
 + Create minimal diffs - disable on save actions like reformat source code or organize imports. If you feel the source code should be reformatted create a separate PR for this change.
-+ Provide JUnit tests for your changes and make sure your changes don't break any existing tests by running ```mvn```.
++ Provide JUnit tests for your changes and make sure your changes don't break any existing tests by running ```mvn clean test```.
```

Comment:
This is not correct, 'mvn' invokes the default maven goal which runs other checks as well.

Reply:
I just merged the generated pages from the 1.x branch and obviously the related change is in there. 

As you may be releasing 2.0 soon, we can also skip this PR.